src/typings/types.d.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Type for Game History variables
+ */
+type GameHistory = Array<{ squares: Array<string> }>;
+
+/**
+ * Type for game state properties
+ */
+type GameStateProps = {
```

Comment:
```suggestion
type GameState = {
```

only append -Props if this type is directly used for component props!